alf/algorithms/predictive_representation_learner.py

```diff
@@ -108,6 +114,7 @@ def calc_loss(self, target, predicted, mask=None):
         Returns:
             LossInfo
         """
+        target = target.view_as(predicted)
```

Comment:
It's shape should be changed at predict_step and train_step so that the output shape matches the target shape.

Reply:
Now changing the shape at train_step and predict_step.